lib/exnumerator.ex

```diff
@@ -11,58 +13,46 @@ defmodule Exnumerator do
       @doc """
       Returns a random value for the enum.
       """
-      def sample, do: unquote(values) |> Enum.random()
+      def sample, do: values() |> Enum.random()
 
       @impl true
       def type, do: :string
 
       @impl true
-      def cast(term)
+      def cast(term) do
+        values() |> Exnumerator.cast(term)
+      end
```

Comment:
I suggest using one-liner syntax for functions with actually one-line body. Will you agree it will reduce unnecessary `end`s there?